fpga_interchange/fasm_generators/nexus/nexus.py

```diff
@@ -16,7 +16,7 @@
 from enum import Enum
 from itertools import product
 
-from fpga_interchange.fasm_generators.generic import FasmGenerator
+from ..generic import FasmGenerator
```

Comment:
Why is this change required?

Reply:
Hmm, It was an attempt to make FASM generators runnable outside the `fpga_interchange` module but it didn't work out. I've reverted the change.